articles/openshift/oshift-ref-no_proxy.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Specifying endpoints to avoid being proxied in deployments with government community network connectivity | UKCloud Ltd
```

Comment:
Can we simplify this title a little bit? Maybe:
Specifying endpoints for government community network connectivity

As long as the detail is in the article, I don't think we need to include it all in the title

Reply:
Community network proxy configuration or something would probably do.